src/Constants.py

```diff
@@ -11,7 +11,7 @@
 LOCAL_HOST = "127.0.0.1"
 EXIT_PAYMENT_TYPE = "exit"
 PROTOCOL_NAME = "hangzhou"
```

Comment:
Do we need this variable? We may need in the future an option to add a custom testnet as e.g. we could want to test on the new "Limanet" temporarily spawning or even Mondaynet? Maybe delete unnecessary information now and add an issue with my request